package.json

```diff
@@ -31,7 +31,7 @@
     "generate:schema": "node ./scripts/generateSchemaTypes.js",
     "docz": "docz",
     "start": "node dist/index.js",
-    "start:development": "ts-node --project ./src/tsconfig.json -r tsconfig-paths/register ./src/index.ts",
+    "start:development": "TS_NODE_PROJECT=./src/tsconfig.json node -r ts-node/register -r tsconfig-paths/register ./src/index.ts",
```

Comment:
Do we need to set `CONCURRENCY=1` here?

Reply:
During development, it's important in my opinion to test like production, so `CONCURRENCY=1` isn't required. The only reason it was added to the vscode `launch.json` is to support actual debugging.